packages/amplify_authenticator/lib/src/l10n/src/inputs/inputs_en.arb

```diff
@@ -107,6 +107,17 @@
             }
         }
     },
+    "warnInvalidFormat": "Not a valid {attributeType} format.",
```

Comment:
nit: I think "Invalid email format" reads better than "Not a valid email format"